docs/contributing/first-time.md

```diff
@@ -163,20 +208,32 @@ Once you have completed, tested, and linted your code, and created a {ref}`contr
     ```shell
     git add <files>
     git commit -m "My commit message"
-    git push
+    git push origin my-branch-name
     ```
 
 1.  Visit your fork of the Plone repository on GitHub, and [create a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) against the development branch.
-    -   Make your title and description descriptive.
-        Don't be lazy with "Fixes bug" or other useless drivel.
-    -   To automatically close the original issue when your pull request is merged, include "Closes #issue_number" in your description.
+    -   Make both your title and description descriptive.
+        Reviewers look at many pull requests, and need to quickly understand the context.
+        A lazily written phrase such as "Fixes bug" is meaningless.
+    -   Include "Fixes #" and the related issue number in the description.
+        This enables automatic closing of the related issue when the pull request is merged.
         This also creates a hyperlink to the original issue for easy reference.
 1.  [Request a review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) from other team members.
 1.  Members who subscribe to the repository will receive a notification and review your request.
 
-```{todo}
-Provide instructions when the contributor needs to update their pull request with changes from the default branch.
-Members of Contributors do not have the button "Update branch" to easily do this, and must use git foo to manage the situation.
+
+### Update your pull request from your fork
+
+Often another pull request will get merged before yours, and your pull request will fall behind the main branch.
+To keep your pull request current, you can issue the follow git commands.
+
+```shell
+# Assume `main` is the main branch.
+git checkout main
+git pull
+git checkout my-branch-name
+git merge main
+git push origin my-branch-name
```

Comment:
I think instead of these commands it's simpler to suggest:
```
# (Assuming you are on the branch for your PR which you want to update)
git fetch upstream  # assumes the upstream remote was configured based on https://docs.github.com/en/get-started/quickstart/fork-a-repo#configuring-git-to-sync-your-fork-with-the-upstream-repository
git merge upstream/main  # You might need to resolve conflicts here.
git push
```

Reply:
I think I got it now, for this suggestion and the next, in 2ec39f33fadf9dd521c4dde5a8dede782041578b. One more review, please?